src/Build.UnitTests/Graph/IsolateProjects_Tests.cs

```diff
@@ -401,7 +401,8 @@ public void UndeclaredReferenceEnforcementShouldNormalizeFilePaths(Func<string,
                     : string.Empty,
                 excludeReferencesFromConstraints
                     ? $"{declaredReferenceFile};{undeclaredReferenceFile}"
-                    : string.Empty)
+                    : string.Empty,
+                NativeMethodsShared.IsMono ? "CodeTaskFactory" : "RoslynCodeTaskFactory")
```

Comment:
Is there a bug that RoslynCodeTaskFactory doesn't work on mono?

Reply:
Removed this. `RoslynCodeTaskFactory` works now.